src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -0,0 +1,71 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.custom-field-input-text {
+  font-family: var(--mavenlink-type-font-family);
+}
+
+.heading-container {
+  color: var(--palette-grey-dark);
+  font-size: var(--mavenlink-type-body-font-size);
+}
+
+.error .heading-container {
+  color: var(--palette-caution-base);
+}
+
+.help {
+  color: var(--palette-grey-dark);
+  font-size: var(--mavenlink-type-base-size);
+}
+
+.error .help {
+  color: var(--palette-caution-base);
+}
+
+.input-container {
+  height: 34px;
+  margin: var(--spacing-small) 0;
+}
+
+input {
```

Comment:
We *cannot* be styling tag names. This will literally style all other inputs on the page as well since it is a generic selector. Instead, use a class name.

Reply:
Ack, nuked the wrong thing! I'll correct this.